### DISCUSSION TITLE
Neuromancer Documentation

### DISCUSSION THREAD

**Discussion by GabSabb** [pnnl/neuromancer#128] [Q&A] [answered]
Hello,

First I would like to thank you for the comprehensive and detailed library that is NeuroMANCER. I usually go to  to the provided piece of [documentation](https://pnnl.github.io/neuromancer/neuromancer.html) to help me navigate the library. As of a couple of weeks ago, the linked documentation seems to have been modified and removed. Will further documentation be discontinued or did it migrate to another domain? 

Thanks in advance!

**Comment by drgona** [comment, marked as answer]
Hi @GabSabb thank you for your interest in our library.

We are currently undergoing a refactoring of the automated documentation, which is currently broken. This will be fixed soon.  

In the meanwhile, the best introduction to the library is our set of google colab notebooks.
https://github.com/pnnl/neuromancer#intro-to-neuromancer
